packages/service-library/src/servicelib/redis.py

```diff
@@ -100,7 +99,10 @@ async def setup(self) -> None:
 
     async def shutdown(self) -> None:
         if self._health_check_task:
-            await stop_periodic_task(self._health_check_task)
+            await stop_periodic_task(
+                self._health_check_task, timeout=_SHUTDOWN_TIMEOUT_S
+            )
+            self._health_check_task = None
 
         # NOTE: redis-py does not yet completely fill all the needed types for mypy
         await self._client.aclose(close_connection_pool=True)  # type: ignore[attr-defined]
```

Comment:
I guess the argument and the mypy exception can be removed since you added this auto_close_connection_pool argument no?

Reply:
mypy exception is still valid the argument can go away